airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/SqlGenerator.java

```diff
@@ -12,6 +12,8 @@ public interface SqlGenerator<DialectTableDefinition> {
 
   ColumnId buildColumnId(String name);
```

Comment:
nit: why not just delete this overload entirely? or at least define a default implementation
```suggestion
  default ColumnId buildColumnId(String name) {
    return buildColumnId(name, "");
  }
```

Reply:
>why not just delete this overload entirely
There's a bunch of test usages that I didn't want to update (and I don't want tests to have to specify "" as the second parameter every time).

>a default implementation
TIL :)